cohortextractor/query_language.py

```diff
@@ -212,14 +211,6 @@ class FilteredTable(BaseTable):
     value: Any
     or_null: bool = False
 
-    def __post_init__(self):
```

Comment:
Do we want to allow filters on just codes?  This was added because the backend `clinical_events` table (for TPP and graphnet at least) has both `code` and `system`, and the query engine builds the codelist temporary table by querying both.  At the time I think Dave said we should require users to specify a system.  Maybe this should be a validation check somewhere else though

Reply:
I think the solution to this will be in the implementation that supports doing `t.filter(t.code in codelist)`.

I hope we'll know then that `t.code` is contains codes.

Reply:
@benbc is there a ticket for error handling?  I think this belongs there.